### ISSUE TITLE
after using the package I'm unable to use widget. to access StatefulWidget attributes

### ISSUE THREAD

**Issue by shadyshrif** [nuruddinsyah/need_resume#1]
This code was working fine

```
class AvailableHomeScreen extends StatefulWidget {
  AppViewModel appViewModel;
  AvailableHomeScreen(this.appViewModel);
  @override
  _AvailableHomeScreenState createState() => _AvailableHomeScreenState();
}

class _AvailableHomeScreenState extends State<AvailableHomeScreen> {
@override
  Widget build(BuildContext context) {
        isSwitched = widget.appViewModel.user.sessionStatus == "available";
       return Text("testing ${isSwitched }");
}
```

And I'm getting an error `The getter 'appViewModel' isn't defined for the type 'StatefulWidget'.\nTry importing the library that defines 'appViewModel'` After I used the `ResumableState` Class instead of `State` Class
```
class AvailableHomeScreen extends StatefulWidget {
  AppViewModel appViewModel;
  AvailableHomeScreen(this.appViewModel);
  @override
  _AvailableHomeScreenState createState() => _AvailableHomeScreenState();
}

class _AvailableHomeScreenState extends ResumableState<AvailableHomeScreen> {
@override
  Widget build(BuildContext context) {
        isSwitched = widget.appViewModel.user.sessionStatus == "available";
       return Text("testing ${isSwitched }");

  @override
  void onResume() {
    // Implement your code inside here
    print('my screen is resumed!');
  }
}
```

**Comment by hdani9307** [comment]
I had the same issue so I fixed it in #2

**Comment by nuruddinsyah** [comment]
Hi @shadyshrif, thanks for your feedback! I forgot to add type cast (`State<T>`).

Hi @hdani9307 big thanks for your fix! I have merged your pull request.

I have updated the library. Check it out: [https://pub.dev/packages/need_resume](https://pub.dev/packages/need_resume) :)
